examples/pytorch/AxHyperOptimizationPTL/iris_data_module.py

```diff
@@ -15,6 +15,9 @@ def __init__(self):
         self.columns = None
 
     def prepare_data(self):
+        pass
```

Comment:
TL;DR
https://github.com/PyTorchLightning/pytorch-lightning/pull/7238 caused the failure. When we call `prepare_data` multiple times, it doesn't return anything after the first call.